kore/test/Test/Kore/Step/Simplification/AndTerms.hs

```diff
@@ -101,6 +102,26 @@ test_andTermsSimplification =
                     )
             actual <- simplifyUnify mkBottom_ fOfA
             assertEqual "" expect actual
+        , testCase "\\and{b}(\\not{b}(\\equals{s, s}(a, b)), \\equals{s, s}(a, b))" $ do
```

Comment:
The `AndTerms` simplifier can't handle unification problems of this type, so I would expect this test to fail (and indeed it does). What did you want to happen here?